etna/lib/etna/server.rb

```diff
@@ -3,7 +3,19 @@ module Etna
   class Server
     class << self
       def route(method, path, options={}, &block)
-        @routes ||= []
+        # For healthchecks, set up servers
+        #   with an OPTIONS route on /, with noauth
+        @routes ||= [
+          Etna::Route.new(
+            'OPTIONS',
+            '/',
+            {
+              auth: {
+                noauth: true
+              }
+            }
+          )
+        ]
```

Comment:
Hmm, won't this conflict with the route introduced by Etna::DescribeRoutes? That layer should trap the request even before Etna::Server sees it. Also we can't disable Etna::DescribeRoutes since it is still used in a bunch of clients unfortunately.

Reply:
It actually does not conflict, and the routes_json is returned. I think you need it as an actual `Etna::Route` on the server, in order for `Etna::Auth` to recognize the `noauth` option. Note that `Etna::DescribeRoutes` does not introduce an `Etna::Route` to the server, it just hijacks the request, so I don't think you can circumvent `Etna::Auth` via `Etna::DescribeRoutes`...

Reply:
So the `Etna::Route` as defined here does not capture / return any data, as you point out, it just allows `Etna::Auth` to do its thing...

Reply:
Perhaps "less testable" isn't quite the right way to word it, but then it's just spread out across all the apps...